R/utilities-simulation.R

```diff
@@ -83,7 +83,8 @@ saveSimulation <- function(simulation, filePath) {
 #' @title  Runs a simulation (individual or population) and returns a \code{SimulationResults} object containing all results of the simulation
 #'
 #' @param simulation Instance of a \code{Simulation} to simulate.
-#' @param population Optional instance of a \code{Population} to use for the simulation
+#' @param population Optional instance of a \code{Population} to use for the simulation.

```

Comment:
@PavelBal I did not rename the parameter as this would break the existing contract for people using explicit naming